### ISSUE TITLE
error with whitenoise

### ISSUE THREAD

**Issue by legshort** [bhch/django-jsonform#44]
with whitenoise environment
`manage.py collectstatic`

after this command, error pops up.

```
whitenoise.storage.MissingFileError: The file 'django_jsonform/local/react-json-form.js.map' could not be found with <whitenoise.storage.CompressedManifestStaticFilesStorage object at 0x112204fd0>.

The JS file 'django_jsonform/local/react-json-form.js' references a file which could not be found:
  django_jsonform/local/react-json-form.js.map
```

thanks in advance.

**Comment by bhch** [comment]
Hello. Thanks for reporting this.

Which version of `django-jsonform` are you using?

Find out with this command:

```
$ python -c "import django_jsonform; print(django_jsonform.__version__)"
```

**Comment by bhch** [comment]
Nevermind, I see the cause of problem now. Related to issue #37.

There's a folder called `local` which I use to keep js files during development. This should not get included in the installable package. But due to some bug in setuptools, the wheel builds don't exclude the `local` folder.

As a temporary solution, you can set [`WHITENOISE_MANIFEST_STRICT = False`](http://whitenoise.evans.io/en/stable/django.html#WHITENOISE_MANIFEST_STRICT) in your settings to suppress this error.

I will fix this issue in the next release. Hopefully by next week.

**Comment by bhch** [comment]
Fixed in [v2.11.0](https://github.com/bhch/django-jsonform/releases/tag/v2.11.0). Please upgrade.
